dev/com.ibm.ws.beanvalidation.v20.cdi/resources/com/ibm/ws/beanvalidation/v20/cdi/internal/BVNLSMessages.nlsprops

```diff
@@ -0,0 +1,46 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# DO NOT TRANSLATE "\n" and "\t" characters.  these are for textual formatting.
+# REMINDER: if message has substitution parameters, then any apostrophe in the message text
+#           must be followed by a 2nd apostrophe to work correctly.
+# -------------------------------------------------------------------------------------------------
+#
+# Briefly:
+#     MESSAGE_ID=<ComponentPrefix><MessageNumber><TypeCode>: <MessageText>
+#
+# Message numbers are four digit numbers, from '0000' to '9999', inclusive.
+# Message numbers are filled to the left with '0', to a width of 4.
+#
+# Valid message type codes are:
+#     I (Informational)
+#     W (Warning)
+#     E (Error)
+#
+# Note the ": " that appears between the type code and the message text.
+#
+# -----------------------------------------------------------------------
+# cmvc_en_version_level=
+#CMVCPATHNAME ws/code/beanvalidation/src/com/ibm/ws/beanvalidation/resources/nls/BVNLSMessages.nlsprops
+#COMPONENTPREFIX CWNBV
+#COMPONENTNAMEFOR CWNBV Bean Validation
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_VAR
+
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} the application name
+#    {1} the exception message from trying to register the CDI extension
+# -----------------------------------------------------------------------
+UNABLE_TO_REGISTER_WITH_CDI=CWNBV0200W: Unable to register Hibernate Validator CDI extension for application {0}. As a result, CDI integration for Bean Validation will not be available for this application. The issue registering the CDI extension was: {1}
+UNABLE_TO_REGISTER_WITH_CDI.explanation=A CDI extension is provided for the built-in Bean Validation provider (Hibernate Validator), which is only supported when the built-in ValidationProvider is used and the validation.xml for the application is valid.  
```

Comment:
Suggested change:
The program provides a CDI extension for the built-in Bean Validation provider, the Hibernate Validator. The program supports the Hibernate Validator only when you use the built-in ValidationProvider class, and the validation.xml file for the application is valid.

Reply:
It sounds awkward saying "the program", can we instead say "Liberty"?  As an end-user, saying "the program" could be ambiguous because it does not identify where in the software stack the issue occurred (e.g. Java, Liberty, some stack product, or the end-user's application)

How about this?
> Liberty provides a CDI extension for the built-in Bean Validation provider (Hibernate Validator). Liberty supports Bean Validation integration with CDI only when you use the built-in Bean Validation providerr, and the validation.xml file for the application is valid.

Reply:
We try to avoid using product names in messages. (Think of how difficult it would be to update if the product name ever changed.) Maybe it's best to use the passive voice:

CDI extension for the built-in Bean Validation provider (Hibernate Validator) is provided. Bean Validation integration with CDI is supported only when you use the built-in Bean Validation provider, and the validation.xml file for the application is valid.

Also, is there a reason for the parenthesis around (Hibernate Validator)? They usually indicate diminished importance, so we generally try to avoid them. However, if this is how they appear in code or something, we can leave the parenthesis. :)

Reply:
Hibernate Validator is name of the implementation we use for our Bean Validation provider, so I agree that it's diminished importance, but still useful information for the end user to know